LML.NPOManagement.Bll/Services/NotificationService.cs

```diff
@@ -201,21 +201,21 @@ private void SendNotification(string body, string subject, string email)
 
 
 
-        public async void PasswordRecoverRequest(UserModel user)
+        public async void PasswordRecoverRequest(UserModel user, string? lang)
         {
             if (user == null)
             {
                 return;
             }
             var userInfo = user.UserInformations.FirstOrDefault();
 
-            var template = await GetTemplateByFileName("RecoverPassword.html");
+            var template = await GetTemplateByFileName("RecoverPassword.html", lang);
 
-            string token = TokenCreationHelper.GenerateJwtToken(user, _configuration, _userRepository);
+            string token = TokenCreationHelper.GenerateJwtToken(user, _configuration, _userRepository, 5);
 
```

Comment:
Roman jan, please explain why are we passing a mogic number 5 into the JWT token helper?